abodepy/event_controller.py

```diff
@@ -46,6 +49,30 @@ def stop(self):
         """Tell the subscription thread to terminate - will block."""
         self._socketio.stop()
 
+    def add_connection_status_callback(self, devices, callback):
+        """Register callback for Abode server connection status."""
+        if not devices:
```

Comment:
What's the purpose of this check? Since `devices` is a required positional argument, how would `add_connection_status_callback` be called without `devices`?

Reply:
This is just a habit I've picked up over the years of null checking all my arguments. In the case that somehow devices is passed in as None, by adding a quick if here I prevent a possible NoneType exception later on.